### PR TITLE
NO JIRA: Add customizing Prometheus page

### PR DESCRIPTION
**Pull request by fredtibbitts**
This PR adds a page to the setup customization section for Prometheus.

### CODE REVIEW

File: content/en/docs/setup/customizing/prometheus.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  components:
+    prometheusOperator:
+      overrides:
+        - values:
+            prometheus:
+              prometheusSpec:
+                replicas: 3
+```
+
+You can enable Alertmanager using the following Verrazzano custom resource:
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/prometheus.md:30]: I would suggest this sentence re-org, "To enable Alertmanager, use the following Verrazzano custom resource:"

**Review by rosemarymarano — APPROVED** [review]
One minor comment. Also, if you wouldn't mind changing AlertManager -> Alertmanager in the 1.3 Release Notes, I'd greatly appreciate it.

File: content/en/docs/setup/customizing/prometheus.md (hunk 1 above)
Comment (fredtibbitts) [reply content/en/docs/setup/customizing/prometheus.md]: Done.

**Comment by fredtibbitts** [comment]
> One minor comment. Also, if you wouldn't mind changing AlertManager -> Alertmanager in the 1.3 Release Notes, I'd greatly appreciate it.

Done.

**Pull request merged**